src/referral.cairo

```diff
@@ -46,11 +57,16 @@ mod Referral {
     //
 
     #[event]
-    fn on_claim(timestamp: u64, amount: u256, sponsor_addr: ContractAddress, ) {}
+    #[derive(Drop, starknet::Event)]
```

Comment:
I thought we had to create a `Event` enum now and then declare a struct for each events, this also works with the latest version of the compiler ?

Reply:
you are right